### PR TITLE
feat:(@desktop/activity-center): AC notification categories according new designs

### LINKED ISSUE #7161: Activity Center categories
Epic: https://github.com/status-im/status-desktop/issues/7070

### Description

The Activity Center must support filtering by thouse categories: 

- [x] All
- [x] Admin (If applies)
- [x] Mentions
- [x] Replies
- [x] Contact Requests
- [x] Identity verification
- [x] Transactions
- [x] Membership
- [x] System

### PR DESCRIPTION
**Pull request by MishkaRogachev**
Close #7161

### What does the PR do

This PR contains is a preparation for new notification categories listed in this epic: *[***](https://github.com/status-im/status-desktop/issues/7070)

### Affected areas

Activity Center

### Screenshot of functionality (including design for comparison)
Screenshot 2022-09-12 at 16 53 14

### CODE REVIEW

File: ui/app/AppLayouts/Chat/panels/ActivityCenterPopupTopBarPanel.qml [hunk 1]
```diff
@@ -9,88 +10,61 @@ import StatusQ.Controls 0.1
...
+            // NO
+            model: [ { text: "All", category: ActivityCenterPopup.ActivityCategory.All, visible: true },
+                     { text: "Admin", category: ActivityCenterPopup.ActivityCategory.Admin, visible: false },
+                     { text: "Mentions", category: ActivityCenterPopup.ActivityCategory.Mentions, visible: false },
+                     { text: "Replies", category: ActivityCenterPopup.ActivityCategory.Replies, visible: false },
+                     { text: "ContactRequests", category: ActivityCenterPopup.ActivityCategory.ContactRequests, visible: false },
+                     { text: "IdentityVerification", category: ActivityCenterPopup.ActivityCategory.IdentityVerification, visible: false },
+                     { text: "Transactions", category: ActivityCenterPopup.ActivityCategory.Transactions, visible: false },
+                     { text: "Membership", category: ActivityCenterPopup.ActivityCategory.Membership, visible: false },
+                     { text: "System", category: ActivityCenterPopup.ActivityCategory.System, visible: false } ]
```
Comment (caybro) [inline ui/app/AppLayouts/Chat/panels/ActivityCenterPopupTopBarPanel.qml:49]: All the `text`s need `qsTr()`

File: ui/app/AppLayouts/Chat/popups/ActivityCenterPopup.qml [hunk 2]
```diff
@@ -188,7 +185,8 @@ Popup {
                             }
                             return -1;
                         }
-                        property string previousNotificationTimestamp: notificationDelegate.idx === 0 ? "" : root.store.activityCenterList.getNotificationData(previousNotificationIndex, "timestamp")
+                        property string previousNotificationTimestamp: notificationDelegate.idx === 0 ? "" :
```
Comment (caybro) [inline ui/app/AppLayouts/Chat/popups/ActivityCenterPopup.qml:188]: `readonly`

**Review by caybro — CHANGES_REQUESTED** [review]

File: ui/app/AppLayouts/Chat/panels/ActivityCenterPopupTopBarPanel.qml (hunk 1 above)
Comment (MishkaRogachev) [reply ui/app/AppLayouts/Chat/panels/ActivityCenterPopupTopBarPanel.qml]: Fixed

File: ui/app/AppLayouts/Chat/popups/ActivityCenterPopup.qml (hunk 2 above)
Comment (MishkaRogachev) [reply ui/app/AppLayouts/Chat/popups/ActivityCenterPopup.qml]: Fixed

File: ui/app/AppLayouts/Chat/panels/ActivityCenterPopupTopBarPanel.qml [hunk 3]
```diff
@@ -9,88 +10,61 @@ import StatusQ.Controls 0.1
...
 
+        StatusListView {
+            id: listView
+            // NO
+            model: [ { text: qsTr("All"), category: ActivityCenterPopup.ActivityCategory.All, visible: true },
+                     { text: qsTr("Admin"), category: ActivityCenterPopup.ActivityCategory.Admin, visible: false },
+                     { text: qsTr("Mentions"), category: ActivityCenterPopup.ActivityCategory.Mentions, visible: false },
+                     { text: qsTr("Replies"), category: ActivityCenterPopup.ActivityCategory.Replies, visible: false },
+                     { text: qsTr("ContactRequests"), category: ActivityCenterPopup.ActivityCategory.ContactRequests, visible: false },
+                     { text: qsTr("IdentityVerification"), category: ActivityCenterPopup.ActivityCategory.IdentityVerification, visible: false },
```
Comment (caybro) [inline ui/app/AppLayouts/Chat/panels/ActivityCenterPopupTopBarPanel.qml:46]: Plus a space between words :)

**Review by caybro — APPROVED** [review]
Looks good from my POV

File: ui/app/AppLayouts/Chat/panels/ActivityCenterPopupTopBarPanel.qml (hunk 3 above)
Comment (MishkaRogachev) [reply ui/app/AppLayouts/Chat/panels/ActivityCenterPopupTopBarPanel.qml]: Fixed

**Pull request merged**